src/Illuminate/Support/helpers.php

```diff
@@ -643,7 +642,7 @@ function env($key, $default = null)
         static $variables;
 
         if ($variables === null) {
-            $variables = (new DotenvFactory([new EnvConstAdapter, new PutEnvAdapter, new ServerConstAdapter]))->createImmutable();
+            $variables = (new DotenvFactory([new EnvConstAdapter, new ServerConstAdapter]))->createImmutable();
```

Comment:
This same line of code is duplicated in `LoadEnvironmentVariables.php`.  You would also need to revert c37702cbdedd4e06eba2162d7a1be7d74362e0cf.